Ports/flare-game/package.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=flare-game
+useconfigure=true
+version=1.14
+depends=(
+  'flare-engine'
+)
+workdir=flare-game-1.14
```

Comment:
Please format `package.sh` according to the [example](https://github.com/SerenityOS/serenity/tree/master/Ports#writing-ports-scripts)

Reply:
Now reformatted :)